aat/src/aat/java/uk/gov/hmcts/ccd/definitionstore/tests/functional/ImportDefinitionTest.java

```diff
@@ -25,6 +25,7 @@ protected ImportDefinitionTest(AATHelper aat) {
 
     private Supplier<RequestSpecification> asUser = asAutoTestCaseworker();
 
+    @Disabled("The response code should be 400 instead of 500. Code needs to be fixed.")
```

Comment:
Have you raised a ticket for this? Have you spoke to developers? 
We can't just comment tests like that

Reply:
https://tools.hmcts.net/jira/browse/RDM-5724 need to be attended later